### ISSUE TITLE
Travis CI: bundle install => double free or corruption (out): 0x00007f607c01e2f0

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1664]
See https://travis-ci.org/jeremyevans/sequel/jobs/520311033

```
Fetching gem metadata from http://rubygems.org/..*** Error in `/home/travis/.rvm/rubies/truffleruby-1.0.0-rc15/bin/truffleruby': double free or corruption (out): 0x00007f607c01e2f0 ***
/home/travis/.travis/functions: line 315:  7315 Aborted                 (core dumped) bundle install --jobs=3 --retry=3

```

**Comment by deepj** [comment]
The same issue in https://travis-ci.org/rom-rb/rom/jobs/521764376

**Comment by eregon** [comment]
I believe I fixed this in the 19.0.0 release, by always recompiling the OpenSSL C extension. Thank you for the report.
